discoveries/coastal-flooding-and-slr.discoveries.mdx

```diff
@@ -0,0 +1,150 @@
+---
+featuredOn:
+  - eis
+id: 'eis-coastal-risks'
+name: Risks to coasts from sea level rise compounded by water cycle intensification
+description: "NASA measurements and models shed light on present and future coastal risks"
+media:
+  src: ::file ./coastal-flooding-and-slr-cover.jpg
+  alt: Flooding in York, UK - February 2020.
+  author:
+    name: Unknown
+    url: https://unsplash.com/photos/P8PuYidUYMw
+pubDate: 2022-11-15
+thematics:
+  - eis
+---
+<Block>
+  <Prose>
+  ## Introduction
+  The Earth Information System (EIS) provides a platform for collaborative analysis and decision support for adapting to sea level changes, human-caused water cycle intensification, and their impacts. The full potential of these individual efforts are realized through the EIS collaborative framework to support new transdisciplinary connections.
+  </Prose>
+</Block>
+
+<Block type='full'>
+  <Figure>
+    <Map
+      datasetId='tws-anomaly'
+      layerId='lis-tws-anomaly'
+      dateTime='2002-12-01'
+      zoom={1}
+      center={[0,0]}
+      compareDateTime='2021-12-01'
+    />
+    <Caption 
+      attrAuthor='NASA' 
+      attrUrl='https://nasa.gov/'
+    >
+      CAPTION
+    </Caption> 
```

Comment:
Caption information is missing here. It is not mandatory to have a caption body. If it is to be removed this can be written as:

```
    <Caption 
      attrAuthor='NASA' 
      attrUrl='https://nasa.gov/'
    />
```

Reply:
Thanks. I removed the CAPTION.